app/controllers/admin/order_cycles_controller.rb

```diff
@@ -86,6 +88,26 @@ def bulk_update
       end
     end
 
+    def bulk_update_nil_subscription_line_items_price_estimate
+      @collection.upcoming.each do |order_cycle|
+        update_nil_subscription_line_items_price_estimate(order_cycle)
+      end
+    end
+
+    def update_nil_subscription_line_items_price_estimate(order_cycle)
+      order_cycle.schedules.each do |schedule|
+        subscription = Subscription.find_by(schedule_id: schedule.id)
```

Comment:
Hm, can't there be more than one subscription per schedule?

Reply:
yeah you are correct. Thanks for pointing that out